Sources/NIOOpenSSL/SSLContext.swift

```diff
@@ -18,7 +18,7 @@ import CNIOOpenSSL
 // This is a neat trick. Swift lazily initializes module-globals based on when they're first
 // used. This lets us defer OpenSSL intialization as late as possible and only do it if people
 // actually create any object that uses OpenSSL.
```

Comment:
just in case, have you run with

- ASan in _release_ mode
- TSan

?

Reply:
Yup, both. All good.